src/integrationTest/java/org/opensearch/test/framework/cluster/LocalCluster.java

```diff
@@ -379,14 +379,23 @@ public Builder nodeSettings(Map<String, Object> settings) {
         }
 
         /**
-        * Adds additional plugins to the cluster
+        * Adds additional plugin to the cluster
         */
         public Builder plugin(Class<? extends Plugin> plugin) {
```

Comment:
nit: We can change the parameter signature here to use spread `Class<? extends Plugin>... plugins` to avoid a n overloaded declaration below.

Something like:

```java
public Builder plugin(Class<? extends Plugin>... plugins) {
    this.plugins.addAll(List.of(plugins));

    return this;
}
```

Reply:
Good idea! Making that update now.